x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/table/attack_details/attack_details_container.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { AttackDiscoveryAlert } from '@kbn/elastic-assistant-common';
+import type { Filter } from '@kbn/es-query';
+import { EuiSpacer, EuiTabs, EuiTab } from '@elastic/eui';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+
+import { AlertsTab } from './alerts_tab';
+import * as i18n from './translations';
+
+/** Tab ID constant for the Attack Summary tab */
+export const ATTACK_SUMMARY_TAB = 'attackSummaryTab';
+/** Tab ID constant for the Alerts tab */
+export const ALERTS_TAB = 'alertsTab';
+
+/** Test subject constant for the tabs container */
+export const TABS_TEST_ID = 'tabs';
+
+interface TabInfo {
+  content: JSX.Element;
+  id: string;
+  name: string;
+}
+
+interface Props {
+  /** The attack discovery alert document. If undefined, only the Alerts tab will be shown. */
+  attack?: AttackDiscoveryAlert;
+  /** Whether to show anonymized values instead of replacements */
+  showAnonymized?: boolean;
+  /** Filters applied from grouping */
+  groupingFilters: Filter[];
+  /** Default filters to apply to the alerts table */
+  defaultFilters: Filter[];
+  /** Whether the alerts table is in a loading state */
+  isLoading: boolean;
```

Comment:
I would consider renaming this prop `isTableLoading` then? That will be a bit more clear, especially once we implement the content of the summary tab, which might have its own loading state?